scripts/workflow.bm.R

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env Rscript
```

Comment:
Is this script needed? Can the changes be added to the normal workflow.R script?